src/Razor/test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/IntegrationTests/ComponentRuntimeCodeGenerationTest/Whitespace_BetweenElementAndFunctions/TestComponent.mappings.txt

```diff
@@ -0,0 +1,9 @@
+Source Location: (36:1,16 [29] x:\dir\subdir\Test\TestComponent.cshtml)
```

Comment:
We generate runtime mappings O_o? We shouldn't be generating mapping baselines for runtime scenarios because they're not valid.

Reply:
How is it invalid exactly? It's not used, but it's not incorrect.

Reply:
We have runtime mappings by accident since we've never added all of the mapping code to the runtime paths :/, sooo anything in a runtime mapping file could be misleading. For instance, in this file it looks like the C# attribute mapping is missing. My bet is that it's due to the differences between runtime and design time. 

If we're making changes to mappings and they impact the runtime mappings it shouldn't matter. Basically, I don't want us to spend time considering file changes that don't actually have a valid impact.

Reply:
This and #line pragmas are totally different things in terms of how our parser treats the two. I'd bet there's still a line pragma on the missing attribute mapping; there's just no generated C# mapping because that information is only ever consumed at design time. 

If we wanted to make the decision that we want to always generate consistent C# mappings for design time and runtime for the sake of simplicity, that's another conversation. However, as it stands today, there's not currently any functional value to having these C# mappings at runtime.

Reply:
Ok point taken.

I'll add a note to the issue tracking the marriage between components and old razor. Basically I had a homegrow a new test infrastructure stack in the Blazor repo, and I just dumped the code in here. That leaves us with a few things to clean up.